Documentation.md

```diff
@@ -0,0 +1,59 @@
+
+## Building and Running
+* [Build Instructions](Base/usr/share/man/man9/BuildInstructions.md)
+* [Advanced Build Instructions](Base/usr/share/man/man9/AdvancedBuildInstructions.md)
+* [Troubleshooting](Base/usr/share/man/man9/Troubleshooting.md)
+* [Running in VirtualBox](Base/usr/share/man/man9/VirtualBox.md)
+* [Running in VMware](Base/usr/share/man/man9/VMware.md)
+
+### OS-specific
+Make sure to read the basic [Build Instructions](Base/usr/share/man/man9/BuildInstructions.md) first.
+* [Building on Windows](Base/usr/share/man/man9/BuildInstructionsWindows.md)
+* [Building on macOS](Base/usr/share/man/man9/BuildInstructionsMacOS.md)
+* [Building on Linux](Base/usr/share/man/man9/BuildInstructionsOther.md)
+
+### Running on Hardware
+* [Bare Metal Installation](Base/usr/share/man/man9/BareMetalInstallation.md)
+* [Running On Raspberry Pi](Base/usr/share/man/man9/RunningOnRaspberryPi.md)
+* [Known Hardware Compatibility](Base/usr/share/man/man9/HardwareCompatibility.md)
+* [Network Booting](Base/usr/share/man/man9/NetworkBoot.md)
+
+## Configuring Editors
+* [CLion](Base/usr/share/man/man9/CLionConfiguration.md)
+* [Emacs](Base/usr/share/man/man9/EmacsConfiguration.md)
+* [Qt Creator](Base/usr/share/man/man9/QtCreatorConfiguration.md)
+* [VS Code](Base/usr/share/man/man9/VSCodeConfiguration.md)
+
+## Development
+* [How to Contribute](CONTRIBUTING.md)
+* [Coding Style](Base/usr/share/man/man9/CodingStyle.md)
+* [Common Patterns](Base/usr/share/man/man9/Patterns.md)
+* [EventLoop](Base/usr/share/man/man7/EventLoop.md)
+* [High DPI Support](Base/usr/share/man/man7/HighDPI.md)
+* [Smart Pointers](Base/usr/share/man/man9/SmartPointers.md)
+* [String Formatting](Base/usr/share/man/man9/StringFormatting.md)
+* [How to Transfer Files Out of Serenity](Base/usr/share/man/man9/TransferringFiles.md)
+
+### File and Data Formats
+* [Application Files (.af)](Base/usr/share/man/man5/af.md)
+* [Bitmap Fonts (.font)](Base/usr/share/man/man5/font.md)
+* [Inter-Process Communication protocol (.ipc)](Base/usr/share/man/man5/ipc.md)
+* [HackStudio Post-Create Scripts (.postcreate)](Base/usr/share/man/man5/postcreate.md)
+* [Clipboard/Drag-and-drop data](Base/usr/share/man/man5/clipboard.md)
+
+## Browser
+* [General Architecture](Base/usr/share/man/man9/Browser-ProcessArchitecture.md)
+* [LibWeb: From Loading to Painting](Base/usr/share/man/man7/LibWeb-FromLoadingToPainting.md)
+* [How to Add an IDL File](Base/usr/share/man/man9/AddNewIDLFile.md)
+
+## Kernel
+* [AHCI Locking](Base/usr/share/man/man9/Kernel-AHCILocking.md)
+* [ProcFS Indexing](Base/usr/share/man/man7/Kernel-ProcFSIndexing.md)
+
+## Miscellaneous Guides
+* [How to Profile the Build](Base/usr/share/man/man9/BuildProfilingInstructions.md)
+* [How to Run Tests](Base/usr/share/man/man9/RunningTests.md)
```

Comment:
:yakshed: Surely everyone tests all of their changes before pushing and forcing the CI to do it, right? ....right? 

I think that ~~the bike shed should be blue, like my bike~~ RunningTests.md should be promoted to ``Building and Running`` and the VMWare/VirtualBox docs should be moved to the "hardware" section, since they depend on the instructions in BareMetalInstallation.md (which used to be called INSTALL.md)